med-inventory/src/layout/NavBar.jsx

```diff
@@ -82,11 +92,27 @@ const useStyles = makeStyles(theme => ({
   },
   toolbar: {
     display: "flex",
-    alignItems: "center",
-    justifyContent: "flex-end",
-    padding: "0 8px",
+    //alignItems: "center",
+    //justifyContent: "flex-end",
+    //padding: "0 8px",
     ...theme.mixins.toolbar
   },
+  toolbarStart:{
+    display : "flex",
+    alignItems : "start",
+    justifyContent : "flex-start"
+  },
+  toolbarEnd:{
+    display : "flex",
+    alignItems : "end",
+    justifyContent : "flex-end",
+    marginLeft :940
```

Comment:
Margin Left value is hard coded. Does this work well in different screen sizes/resolutions?

Reply:
I will do it with Flex and update